run/helm/snap/templates/config/_task.tpl

```diff
@@ -0,0 +1,19 @@
+---
+max-failures: 10
```

Comment:
OK, I agree. I remove task template as it is not needed here and I as to _snapteld.conf I will leave it partly templatable.